multiversx_sdk/abi/managed_decimal_value_test.py

```diff
@@ -0,0 +1,41 @@
+from decimal import Decimal
+
+from multiversx_sdk.abi.managed_decimal_value import ManagedDecimalValue
+
+
+class TestManagedDecimalValueTest:
```

Comment:
Maybe have a single test file to test both signed and unsigned values?

Also, maybe have some unit tests around encoding and decoding, as well?

Reply:
now having the tests in the same file. did some encoding and decoding tests, as well.